codepropertygraph/src/main/scala/io/shiftleft/codepropertygraph/cpgloading/ProtoToCpg.scala

```diff
@@ -0,0 +1,145 @@
+package io.shiftleft.codepropertygraph.cpgloading
+
+import java.lang.{Boolean => JBoolean, Integer => JInt, Long => JLong}
+import java.util.{NoSuchElementException, Collection => JCollection, Iterator => JIterator}
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.proto.cpg.Cpg.CpgStruct.{Edge, Node}
+import io.shiftleft.proto.cpg.Cpg.PropertyValue
+import org.apache.commons.configuration.Configuration
+import org.apache.logging.log4j.{LogManager, Logger}
+import org.apache.tinkerpop.gremlin.structure.{T, Vertex}
+import org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerGraph
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ArrayBuffer
+
+object ProtoToCpg {
+  val logger: Logger = LogManager.getLogger(classOf[ProtoToCpg])
+
+  def addProperties(kvs: ArrayBuffer[AnyRef], name: String, value: PropertyValue): Unit = {
+    import io.shiftleft.proto.cpg.Cpg.PropertyValue.ValueCase._
+    value.getValueCase match {
+      case INT_VALUE =>
+        kvs += name
+        kvs += (value.getIntValue: JInt)
+      case STRING_VALUE =>
+        kvs += name
+        kvs += value.getStringValue
+      case BOOL_VALUE =>
+        kvs += name
+        kvs += (value.getBoolValue: JBoolean)
+      case STRING_LIST =>
+        value.getStringList.getValuesList.asScala.foreach { elem: String =>
+          kvs += name
+          kvs += elem
+        }
+      case VALUE_NOT_SET => ()
+      case _ =>
+        throw new RuntimeException("Error: unsupported property case: " + value.getValueCase.name)
+    }
+  }
+}
+
+class ProtoToCpg(val overflowConfig: OverflowDbConfig = OverflowDbConfig.withDefaults) {
+  private val nodeFilter: NodeFilter = new NodeFilter
+  private val tinkerGraph: TinkerGraph = mkTinkerGraph
+
+  import ProtoToCpg._
+
+  private def mkTinkerGraphConfig: Configuration = {
```

Comment:
Please rename to `makeTinkerGraphConfig` or `createTinkerGraphConfig` to avoid abbreviations like above.